migration-guides/11.0.md

```diff
@@ -29,7 +29,7 @@ or by disabling it for your generator in your `openapitools.json` file:
 {
   "$schema": "https://raw.githubusercontent.com/OpenAPITools/openapi-generator-cli/master/apps/generator-cli/src/config.schema.json",
   "generator-cli": {
-    "version": "7.4.0", // Version of the Codegen downloaded by the cli - updated via the 
+    "version": "7.4.0", // Version of the Codegen downloaded by the cli - updated via the
```

Comment:
sentence is not finished

Reply:
Not my sentence but in the doubt, I removed the unfinished part